static/js/tools/stat_var/page.tsx

```diff
@@ -66,21 +93,32 @@ class Page extends Component<unknown, PageStateType> {
     });
   }
 
+  private filterStatVars = (entities: NamedPlace[]) => {
+    this.setState({ entities });
+  };
+
   render(): JSX.Element {
+    const svs = this.state.statVar ? { [this.state.statVar]: {} } : {};
     return (
       <>
         <StatVarWidget
           openSvHierarchyModal={this.state.showSvHierarchyModal}
           openSvHierarchyModalCallback={this.toggleSvHierarchyModal}
           collapsible={false}
           svHierarchyType={StatVarHierarchyType.STAT_VAR}
-          samplePlaces={[]}
+          samplePlaces={this.state.entities}
           deselectSVs={() => this.updateHash("")}
-          selectedSVs={{ [this.state.statVar]: {} }}
+          selectedSVs={svs}
           selectSV={(sv) => this.updateHash(sv)}
+          disableAlert={true}
```

Comment:
I'm curious why we want to disable the alert for the stat var page, because wouldn't we want to alert the user when a variable is deselected?

Reply:
this use case seemed a little different than the other tools to me, because a user is deliberately filtering the stat vars here (rather than picking a place without knowing whether a stat var exists). so it seemed expected that a stat var the user is currently looking at might not exist for a completely different source (and in fact, given the distribution of stat vars vs sources, this actually happens most of the time). because of this, when I was testing, it seemed a bit inconvenient to have to keep closing the alert, so instead it'll just redirect to the stat var landing page. wdyt?

Reply:
ok, that makes sense to me!